docs/breaking-changes/v6.md

```diff
@@ -0,0 +1,24 @@
+# Breaking changes in v6
+
+Branches configuration is now simplified and only `main` and maintenance branches are supported.
+
+We no longer support pre releases for feature branches.
+
+## Upgrade instructions
+
+If you still need to support pre releases for feature branches, you can add the following configuration
+to your semantic-release config file:
+
+```json
+{
+  "branches": [
+    "main",
+    "+([0-9])?(.{+([0-9]),x}).x",
+    {
+      "name": "(f|b|c)/*",
+      "channel": "next",
+      "prerelease": "beta-${(/^[a-zA-Z]+-[0-9]+/.exec(name.substr(2)) || [name.replace(/[_/.]/g, '-')])[0]}-${Date.now()}"
+    }
+  ]
+}
```

Comment:
so we'd have to add this back to each of the FE libraries (e.g. web-ui, node-urls, etc) to keep ability of releasing beta versions?

Reply:
No, we would have to adopt the `prerelease` workflow into those repos and things will still work the same way

Because our actions are not pinning the version, we would have to adopt that pattern internally or update the semantic-release config while we adopt them as you say until we have fully migrated